pandas/tests/tseries/offsets/conftest.py

```diff
@@ -2,9 +2,59 @@
 
 import pandas.tseries.offsets as offsets
 
+DATE_OFFSETS = {
+    "Day",
+    "MonthBegin",
+    "MonthEnd",
+    "SemiMonthBegin",
+    "SemiMonthEnd",
+    "YearBegin",
+    "YearEnd",
+    "QuarterBegin",
+    "QuarterEnd",
+    "LastWeekOfMonth",
+    "Week",
+    "WeekOfMonth",
+    "Easter",
+    "Hour",
+    "Minute",
+    "Second",
+    "Milli",
+    "Micro",
+    "Nano",
+    "DateOffset",
+}
 
-@pytest.fixture(params=[getattr(offsets, o) for o in offsets.__all__])
-def offset_types(request):
+BUSINESS_OFFSETS = {
+    "BusinessDay",
+    "BDay",
+    "CustomBusinessDay",
+    "CBMonthBegin",
+    "CBMonthEnd",
+    "BMonthBegin",
+    "BMonthEnd",
+    "BusinessHour",
+    "CustomBusinessHour",
+    "BYearBegin",
+    "BYearEnd",
+    "CDay",
+    "BQuarterBegin",
+    "BQuarterEnd",
+    "FY5253Quarter",
+    "FY5253",
+}
+
+
+@pytest.fixture(params=[getattr(offsets, o) for o in DATE_OFFSETS])
+def date_offset_types(request):
+    """
+    Fixture for all the datetime offsets available for a time series.
+    """
+    return request.param
+
+
+@pytest.fixture(params=[getattr(offsets, o) for o in BUSINESS_OFFSETS])
+def business_offset_types(request):
     """
     Fixture for all the datetime offsets available for a time series.
```

Comment:
same docstring for business_offset_types as date_offset_types docstring. should differentiate.

Reply:
Added in #31034